webassembly/F3DEmscriptenBindings.cxx

```diff
@@ -346,7 +346,9 @@ EMSCRIPTEN_BINDINGS(f3d)
       emscripten::return_value_policy::reference())
     .function("stop", &f3d::interactor::stop, emscripten::return_value_policy::reference())
     .function("requestRender", &f3d::interactor::requestRender,
-      emscripten::return_value_policy::reference());
+      emscripten::return_value_policy::reference())
+    .function(
+      "requestStop", &f3d::interactor::requestStop, emscripten::return_value_policy::reference());
```

Comment:
add (smoke) test

Reply:
@Meakk how do I go about this?

Reply:
see webassembly/testing/test_interactor.js:73

Reply:
imo thats enough to ensure the binding is not removed. I'll defer to @Meakk if we need more.